src/tagging-compliance/resource_parsing.py

```diff
@@ -111,4 +111,4 @@ def lambda_handler(event, context):
     push_to_gateway(
         os.environ["prometheus_ip"], job="TaggingResourceList", registry=registry
     )
-    return {"statusCode": 200, "body": "success"}
+    return {"statusCode": 200, "body": "success"}
```

Comment:
EOF missing

Reply:
Added in d0ff7d06072eb4c290cb31d2d612c49dd5c6e6ed